bsp/m5dial/include/bsp/m5dial.h

```diff
@@ -0,0 +1,230 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief ESP BSP: M5Stack CoreS3
+ */
+
+#pragma once
+
+#include "sdkconfig.h"
+#include "driver/gpio.h"
+#include "driver/i2c.h"
+#include "bsp/config.h"
+#include "bsp/display.h"
+
+#if (BSP_CONFIG_NO_GRAPHIC_LIB == 0)
+#include "lvgl.h"
+#include "esp_lvgl_port.h"
+#endif // BSP_CONFIG_NO_GRAPHIC_LIB == 0
+
+/**************************************************************************************************
+ *  BSP Capabilities
+ **************************************************************************************************/
+
+#define BSP_CAPS_DISPLAY        1
+#define BSP_CAPS_TOUCH          1
+#define BSP_CAPS_BUTTONS        0
```

Comment:
We use button and knob components here

```suggestion
#define BSP_CAPS_BUTTONS        1
#define BSP_CAPS_KNOB              1
```